protoc_plugin/plugin.py

```diff
@@ -86,9 +84,9 @@ def render(
                     name, cardinality, request_type, reply_type = method
                     wrapper_cls: type
                     if cardinality is const.Cardinality.UNARY_UNARY:
-                        wrapper_cls = UnaryUnaryWrapper
+                        wrapper_cls = "modal._utils.grpc_utils.UnaryUnaryWrapper"
```

Comment:
Do we still need `modal._utils.grpc_utils` imported under a `TYPE_CHECKING` guard to use a forward reference? Never been totally clear on that.

Reply:
In this case it's not a forward reference - it's a "real" reference in the generated code and it's added as an import on line 175: https://github.com/modal-labs/modal-client/pull/2181/files#diff-d721170dbb2b36a3f20394f9563415ebe89a0916e4957cf99e7b615cfd8c772fR175

In case of forward/str references I think the TYPE_CHECKING-guarded imports are still required for type checkers to know what it's dealing with